src/main/flight/feedforward.c

```diff
@@ -93,22 +93,25 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
 
         // interpolate setpoint if necessary
         if (rcCommandDelta == 0.0f) {
-            if (prevDuplicatePacket[axis] == false) {
-                // first duplicate after movement, interpolate setpoint and use previous acceleration
-                // don't interpolate if sticks close to centre or max, interpolate jitter signals less than larger ones
-                if (setpointPercent > 0.02f && setpointPercent < 0.95f) {
-                    // setpoint interpolation includes previous acceleration and attenuation
-                    setpoint = prevSetpoint[axis] + (prevSetpointSpeed[axis] + prevAcceleration[axis] * jitterAttenuator ) * rxInterval * jitterAttenuator;
-                    // recalculate speed and acceleration
-                    setpointSpeed = (setpoint - prevSetpoint[axis]) * rxRate;
-                }
+            // duplicate packet data on this axis
+            if (goodPacketCount[axis] == 2 && setpointPercent < 0.95f) {
```

Comment:
Safer to use `>=` 2 here.

Reply:
OK though it shouldn't be able to get >2, but I'll change, no problem.

Reply:
It will not go above 2 _now_. But coding this defensively so that this will still work even if a future code change makes it possible for this to go higher makes this more robust, and less likely to lead to hard to spot bugs.